src/WebClient.spec.js

```diff
@@ -191,6 +191,19 @@ describe('WebClient', function () {
       });
     });
 
+    it('should return platform error if 200 request is not ok', function () {
+      const scope = nock('https://slack.com')
+        .post(/api/)
+        .reply(200, { ok: false });
+      const client = new WebClient(token);
+      return client.apiCall('method')
+        .catch((error) => {
+          assert.propertyVal(error, 'code', 'slackclient_platform_error');
+          assert.propertyVal(error.data, 'ok', false);
+          scope.done();
```

Comment:
we're missing an assertion on `error.data.error`, which should be equal to the `error` property in the response (as commented above).

Reply:
taken care of in latest commit; along with `error: 'bad error'` to line 197, also added `assert.nestedPropertyVal(error, 'data.error', 'bad error')` to inner function of the `catch`